frontend/apps/reader/modules/readersearch.lua

```diff
@@ -39,6 +39,9 @@ function ReaderSearch:onShowFulltextSearchInput()
 
     self.input_dialog = InputDialog:new{
         title = _("Enter text to search for"),
+        input = self.last_fulltext,
+        check_button_text = self.ui.document.provider == "crengine" and _("Use regular expression"),
```

Comment:
Not fond of seeing `self.ui.document.provider == "crengine"` to distinguish between epub/rolling and pdf/paging.
Usually, we use `if self.ui.document.info.has_pages then` for that - but testing for `self.ui.rolling` is even simpler (rolling means we're using crengine, there's too many things dependant on credocument in readerrolling that we'll never use it with any kind of other rolling engine :)

Reply:
OK